src/pipx/venv.py

```diff
@@ -327,8 +327,8 @@ def list_installed_packages(self) -> Set[str]:
         pip_list = json.loads(cmd_run.stdout.strip())
         return set([x["name"] for x in pip_list])
 
-    def run_app(self, app: str, app_args: List[str]) -> NoReturn:
-        exec_app([str(self.bin_path / app)] + app_args)
+    def run_app(self, app: str, filename: str, app_args: List[str]) -> NoReturn:
+        exec_app([str(self.bin_path / filename)] + app_args)
```

Comment:
`app` doesn't seem to be used in this function at all anymore.

Reply:
This argument will be needed to query the `[pipx.run]` hook so I leave it in for now.